models/gpt2/model.py

```diff
@@ -0,0 +1,196 @@
+from os import path
+import functools
```

Comment:
Please add license headers to all source files

Also check that the license header style is consistent across the project. I see two different versions:

https://github.com/google/iree-jax/blob/b26ae072cc80f8d2fa9e90c26d790d3b3a624c03/setup.py#L1-L5

https://github.com/google/iree-jax/blob/b26ae072cc80f8d2fa9e90c26d790d3b3a624c03/iree/jax/builtins.py#L1-L13

(and it looks like iree-jax doesn't have the LLVM exception set right now: https://github.com/google/iree-jax/blob/main/LICENSE, compared to https://github.com/google/iree/blob/main/LICENSE, so the longer form might be correct?)

Reply:
Changed to consistent version and used long form version.